diff --git a/CMakeLists.txt b/CMakeLists.txt
index 65a8811..446d76a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -227,7 +227,6 @@ add_library(SDL2_image::${sdl2_image_export_name} ALIAS SDL2_image)
 target_include_directories(SDL2_image PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
     "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
-    "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>"
 )
 target_compile_definitions(SDL2_image PRIVATE
     BUILD_SDL
@@ -741,7 +740,7 @@ if(SDL2IMAGE_INSTALL)
     )
     install(
         FILES "${CMAKE_CURRENT_SOURCE_DIR}/include/SDL_image.h"
-        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/SDL2" COMPONENT DEVEL
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/SDL2_image" COMPONENT DEVEL
     )
 
     if(INSTALL_EXTRA_TARGETS)
diff --git a/include/SDL_image.h b/include/SDL_image.h
index 138fcbf..fda91dc 100644
--- a/include/SDL_image.h
+++ b/include/SDL_image.h
@@ -29,9 +29,9 @@
 #ifndef SDL_IMAGE_H_
 #define SDL_IMAGE_H_
 
-#include "SDL.h"
-#include "SDL_version.h"
-#include "begin_code.h"
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_version.h>
+#include <SDL2/begin_code.h>
 
 /* Set up for C function definitions, even when using C++ */
 #ifdef __cplusplus
@@ -2188,6 +2188,6 @@ extern DECLSPEC IMG_Animation * SDLCALL IMG_LoadWEBPAnimation_RW(SDL_RWops *src)
 #ifdef __cplusplus
 }
 #endif
-#include "close_code.h"
+#include <<SDL2/close_code.h>
 
 #endif /* SDL_IMAGE_H_ */
